Makefile

```diff
@@ -1,10 +1,19 @@
-.PHONEY: binary
+.PHONY: all binary ut clean
 
-binary: dist/calico
+BUILD_DIR=build_calico_rkt
+BUILD_FILES=Dockerfile requirements.txt
 
-dist/calico:
-	# Build docker container
-	cd build_calico_rkt; docker build -t calico-rkt-build .
+default: all
+all: binary test
```

Comment:
binary isn't required here.  You can follow the path of test and see that it makes sure the binary is installed before continuing.